process/subprocesses/_10_gtfs_analysis.py

```diff
@@ -326,39 +319,43 @@ def gtfs_analysis(codename):
                         print(
                             f'  - configured {mode} agency id numbers: {agency_ids}',
                         ),
-                stops_headway = _gtfs_utils.get_hlc_stop_frequency(
-                    loaded_feeds,
-                    start_hour,
-                    end_hour,
-                    start_date,
-                    end_date,
-                    route_types,
-                    agency_ids,
-                    dow=dow,
-                    frequencies=frequencies_df,
-                )
-
-                stop_count = len(stops_headway)
-                all_stop_count = len(mode_stops)
-                if stop_count > 0:
-                    stop_frequent_final = pd.merge(
-                        loaded_feeds.stops[
-                            (loaded_feeds.stops['stop_id'].isin(mode_stops))
-                        ],
-                        stops_headway,
-                        how='left',
-                        on='stop_id',
-                    )
-                    stop_frequent_final['authority'] = feed['gtfs_provider']
-                    stop_frequent_final['mode'] = mode
-                    stop_frequent_final['feed'] = gtfs_feed
-                    stop_frequent = pd.concat(
-                        [stop_frequent, stop_frequent_final],
-                        ignore_index=True,
+                    stops_headway = _gtfs_utils.get_hlc_stop_frequency(
+                        loaded_feeds,
+                        start_hour,
+                        end_hour,
+                        start_date,
+                        end_date,
+                        route_types,
+                        agency_ids,
+                        dow=dow,
                     )
-                if all_stop_count > 0:
+
```

Comment:
This 'all_stop_count' variable was idnetifical to 'mode_stops_count', and used within a loop where mode_stop_count > 0.  Therefore this conditional loop could just be indented as by definition, all_stop_count (aka mode_stops_count) was already > 0, and should be actioned in that loop